gui/test/integration/cypress/support/page_files/Page.ts

```diff
@@ -9,17 +11,8 @@ export abstract class Page {
 
   readonly frame = {
     navbar: {
-      get nodeLabelSearchInput() {
-        return cy.get(
-          "[data-cy=frame] [data-cy=navbar] [data-cy=searchTextInput]"
-        );
-      },
-
-      search(text: string) {
-        const field = this.nodeLabelSearchInput;
-        field.clear();
-        field.type(text + "{enter}");
-        return this;
+      get search() {
+        return new NodeSearchInput("[data-cy=frame] [data-cy=navbar]");
```

Comment:
It's good you've replaced this. I prefer the sub-objects take their own selector rather than a prefix, though.

Reply:
Clarified on slack